### DISCUSSION TITLE
Hiding previous and next links

### DISCUSSION THREAD

**Discussion by ZuluPilgrim** [GetPublii/Publii#1451] [Help] [answered]
How do I remove the previous and next links from the bottom of a page?

![image](https://github.com/GetPublii/Publii/assets/111164245/af7e47aa-051f-430a-b101-306ab1a8f022)

**Comment by 3muchfun** [comment, marked as answer]
Try disabling "Display post pagination"  under Custom settings. )It is depending of the theme you use, whether this setting is visible or not). 
<img width="759" alt="Bildschirmfoto 2024-03-22 um 22 52 13" src="https://github.com/GetPublii/Publii/assets/17292904/052fa98b-0cc2-4aa0-8744-92f30b1c6be0">

**Reply by ZuluPilgrim** [reply]
That did it. 

Many thanks :)
